src/scraper/items.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+
+# Define here the models for your scraped items
+#
+# See documentation in:
+# http://doc.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class ScraperItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
+    pass
```

Comment:
These files are autogenerated and we didn't use them anywhere yet, but as far as I know they are required for scrapy to be able to correctly run the project (which is also why they're included).